### PR TITLE
DDL - Data Types

### PR DESCRIPTION
**Pull request by mihaiberq**

### CODE REVIEW

**Comment by enki-hq** [comment]
- [ ] https://enkipro.com/preview-insight/sagelabs/content/SQL/DDL/data-types/handling-numbers.md?ref=83771135ee40e7f370e8e027a0a70fe30acd0699

File: SQL/DDL/data-types/handling-numbers.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# Handling Numbers
```
Comment (lizTheDeveloper) [inline SQL/DDL/data-types/handling-numbers.md]: Change title to SQL Numeric Data Types (so the revision question looks good)

File: SQL/DDL/data-types/handling-numbers.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+Smaller integers are as follows:
+- `tinyint`: 8bit/1 bytes (MySQL, MSSQL)
+- `smallint`: 16bit/2 bytes (PSQL, MySQL, MSSQL)
+
+Bigger integers are as follows:
+- `bigint`: 64bit/8 bytes (PSQL, MySQL, MSSQL)
+
+### Serials
+
+Available only in PSQL and MySQL, serials are unsigned auto-incrementing integers. MySQL's `serial` defaults to 8 bytes values (`bigint`), while in PSQL all three variants are available (`smallserial`, `serial`, `bigserial`).
```
Comment (lizTheDeveloper) [inline SQL/DDL/data-types/handling-numbers.md]: Add a note that if you are using Serial as IDs, use `bigint`, with a footnote to this link:
https://hackernoon.com/the-night-the-postgresql-ids-ran-out-9430a2dbb895

File: SQL/DDL/data-types/handling-numbers.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+- MSSQL has no special `boolean` type, but its behavior can be simulated by `bit(1)`, with 0 for false and 1 for true
+
+### Money
+
+A datatype used to store currency values:
+- `money`: 64bit/8 byte - supports 2 decimals for PSQL and 4 decimals for MSSQL
+- MySQL has no currency data type, but `decimal(n, d)` can be used to simulate the behavior
+
+---
+## Practice
```
Comment (lizTheDeveloper) [inline SQL/DDL/data-types/handling-numbers.md]: make this the revision q

Comment (lizTheDeveloper) [reply SQL/DDL/data-types/handling-numbers.md]: for the practice, have them create a table with a serial ID column that's bigint

**Comment by enki-hq** [comment]
bad indentation of a sequence entry at line 3, column 28:
      - [PostgreSQL Data Types](https://www.postgresql.org/docs ... 
                               ^

**Comment by enki-hq** [comment]
bad indentation of a sequence entry at line 3, column 28:
      - [PostgreSQL Data Types](https://www.postgresql.org/docs ... 
                               ^

File: SQL/DDL/data-types/encoding-and-collation.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+### Collation
+
+**Collation** means collecting and combining pieces of written information. It is the reason why 'é' (whose ASCII code point is 130) is seen as another type of 'e' and not something that comes after 'z' (122 in ASCII) in a clause that sorts results by a string field.
+
+For this reason, each language has its own collation rules. Most of the time, the table default collation rule is deduced based on the locale settings (`en_US`, `fr_FR`, etc.). However, specific ones can be set for columns and even operations.
+
+---
+## Practice
+
+As of now, the most used character encoding is
```
Comment (lizTheDeveloper) [inline SQL/DDL/data-types/encoding-and-collation.md]: oh man- this is _hotly_ debated. I would say the most _reccomended_.

**Comment by lizTheDeveloper** [comment]
💯💫✨🌈⚡️ is what I have to say about this!
Way to go- those two workouts instead of those big insights are much more complete, much deeper, and much more valuable! The questions are great too- they're all deeper content that the original insight could be alone. Now we have a bank of questions that really get at whether or not someone knows their SQL server differences, and how to interact with blobs and json.

**Pull request merged**